src/tests/ftest/pool/rebuild_tests.py

```diff
@@ -16,6 +16,9 @@ class RebuildTests(TestWithServers):
     :avocado: recursive
     """
 
+    def __init__(self):
```

Comment:
(pylint-super-init-not-called) __init__ method from base class 'TestWithServers' is not called